virttest/utils_misc.py

```diff
@@ -3470,6 +3470,30 @@ def check_device_driver(pci_id, driver_type):
     return driver == driver_type
 
 
+def get_guest_os_datetime(vm, os_type):
```

Comment:
view from you function name, this function should be return a float or a datetime type values.